public_html/about.php

```diff
@@ -68,6 +65,15 @@
     $contributors_html .= '</div>';
     // Close card div
     $contributors_html .= '</div></div>';
+
+    // Location JSON
+    if(array_key_exists('location', $c)){
+        $location['location'] = $c['location'];
+        $location['full_name'] = array_key_exists('full_name', $c) ? $c['full_name'] : '';
+        $location['image_fn'] = $img_path;
```

Comment:
We should include also `array_key_exists('image_fn')` for an image, because right now The Competence Centre for Genome Analysis Kiel is missing one.

Reply:
Good spot 👍